db/seeds.rb

```diff
@@ -8,79 +8,359 @@
 
 require 'faker'
 
+puts "Deleting all tables..."
```

Comment:
Seeds shouldn't delete data (ever). Please remove this section. Any deletions in development have to be done manually to avoid accidentally deleting important data in a production environment.